linode/terraform-provider-linode
# Update linodego for DC Get Well

## 📝 Description

The latest [linodego](https://github.com/linode/linodego/releases/tag/v1.26.0) version has been released including the DC get well change. We can now revert the linodego replace and update the version. 

## ✔️ How to Test

Run integration tests against production:

`make PKG_NAME="linode/accountavailability" testacc`
`make PKG_NAME="linode/accountavailabilities" testacc`

**Manual test:**

1. In a provider sandbox environment (e.g. dx-devenv), apply the following configuration:
```
data "linode_account_availability" "foobar" {
    region = "us-east"
}

output "account_availability" {
    value = data.linode_account_availability.foobar.*
}
```

2. Apply the terraform config and observe the output change. Should be similar as:
```
Outputs:
account_availability = [
  {
    "region" = "us-east"
    "unavailable" = tolist([])
  },
]
```
